x/twap/types/utils.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/osmosis-labs/osmosis/v11/osmoutils"
 )
 
+var MaxSpotPrice = sdk.NewDec(2).Power(128).Sub(sdk.OneDec())
```

Comment:
what's the reason we don't import it / use it directly from gamm module but declare it twice both in twap module and gamm module?

Reply:
imports basically, introduces a dependency on its types package. (Maybe this doesn't make sense though)

I think it is ok for the two to diverge with time